nx/lib/nx.ex

```diff
@@ -3097,33 +3163,137 @@ defmodule Nx do
         ]
       >
 
+  ## Vectorized tensors
+
+  Vectors remain unchanged, and normal broadcast rules apply otherwise.
+
+      iex> a = Nx.tensor([[[1, 2, 3]], [[4, 5, 6]]]) |> Nx.vectorize(:x)
+      iex> Nx.broadcast(a, {2, 3})
+      #Nx.Tensor<
+        vectorized[x: 2]
+        s64[2][3]
+        [
+          [
+            [1, 2, 3],
+            [1, 2, 3]
+          ],
+          [
+            [4, 5, 6],
+            [4, 5, 6]
+          ]
+        ]
+      >
+
+  For tensor inputs, the broadcast will only take the shape in consideration.
```

Comment:
```suggestion
  For tensor shapes, the broadcast will only take the shape in consideration.
```

Reply:
I meant tensors as inputs in the second argument. Which also answers the question below regarding vectorizing with `:a` as well.

Reply:
I understood that. But it is a bit unclear because everything has input tensors. If we want to be clearer, we could say “tensors as shapes”?

Reply:
Yeah, that's better!